emri_sur_load.py

```diff
@@ -119,32 +119,57 @@ def alpha_scaling_time(q):
+def slog_surrogate(q, h_eim_amp_spline, h_eim_ph_spline, eim_indicies_amp, eim_indicies_ph, B_amp, B_ph, calibrated):
     """ Compute the interpolated waveform for a single mode """
     h_eim_amp = np.array([splev(np.log(q), h_eim_amp_spline[j])  for j in range(len(eim_indicies_amp))])
     h_eim_ph = np.array([splev(np.log(q), h_eim_ph_spline[j]) for j in range(len(eim_indicies_ph))])
     h_approx_amp = np.dot(B_amp.transpose(), h_eim_amp)
     h_approx_ph = np.dot(B_ph.transpose(), h_eim_ph)
     h_approx = amp_ph_to_comp(h_approx_amp, h_approx_ph)
-    h_approx = alpha_scaling_h(q,h_approx)*(1/q) # because the training waveform follows definition q<1 and we follow q>1
-    return h_approx
+    if calibrated==True:
+        h_approx = alpha_scaling_h(q,h_approx) # because the training waveform follows definition q<1 and we follow q>1
+    return np.array(h_approx)*(1/q)
 
-def surrogate(modes, q_input, h_eim_amp_spline, h_eim_ph_spline, eim_indicies_amp, eim_indicies_ph, B_amp, B_ph):
+def surrogate(modes, q_input, h_eim_amp_spline, h_eim_ph_spline, eim_indicies_amp, eim_indicies_ph, B_amp, B_ph, calibrated):
     """ Takes the interpolation indices, spline nodes, matrix B and computes the interpolated waveform for all modes"""
     h_approx={}
     for mode in modes:
-        h_approx[(mode)] = slog_surrogate(q_input, h_eim_amp_spline[(mode)], h_eim_ph_spline[(mode)], eim_indicies_amp[(mode)], eim_indicies_ph[(mode)], B_amp[(mode)], B_ph[(mode)])
+        h_approx[(mode)] = slog_surrogate(q_input, h_eim_amp_spline[(mode)], h_eim_ph_spline[(mode)], eim_indicies_amp[(mode)], eim_indicies_ph[(mode)], B_amp[(mode)], B_ph[(mode)], calibrated)
         h_approx[(mode)] = np.array(np.conj(h_approx[(mode)])) # needed to match convention of other surrogate models
         
     return h_approx
 
-def generate_surrogate(q_input,modes=[(2,1),(2,2),(3,1),(3,2),(3,3),(4,2),(4,3),(4,4),(5,3),(5,4),(5,5)]):
+def generate_surrogate(q_input,modes=[(2,1),(2,2),(3,1),(3,2),(3,3),(4,2),(4,3),(4,4),(5,3),(5,4),(5,5)], calibrated=True):
     """ Top-level function to evaluate surrogate waveform"""
     
-    h_approx = surrogate(modes,q_input, h_eim_amp_spline, h_eim_ph_spline, eim_indicies_amp, eim_indicies_ph, B_amp, B_ph)
-    time_approx=alpha_scaling_time(q_input)
-    return time_approx, h_approx
-
+    h_approx = surrogate(modes,q_input, h_eim_amp_spline, h_eim_ph_spline, eim_indicies_amp, eim_indicies_ph, B_amp, B_ph, calibrated)
+    if calibrated==True:
+        time_approx=alpha_scaling_time(q_input)
+        return time_approx, h_approx
+    else:
+        return np.array(time), h_approx
+
+def generate_surrogate_physical(q_input,modes=[(2,1),(2,2),(3,1),(3,2),(3,3),(4,2),(4,3),(4,4),(5,3),(5,4),(5,5)], M_total=80, dis=100):
+    """ Top-level function to evaluate surrogate waveform in physical units"""
+    
+    time_approx, h_approx = generate_surrogate(q_input, modes, calibrated=True)
+    
+    # Physical units
+    G=6.674*1e-11
```

Comment:
these constants can be gotten from gwtools. The benefit of this is it will make it easier to maintain consistency between gwsurrogate and the BHPTK implementations if the "constants" ever change. G is likely to change in the future, for example. 

at the top of this code do `from gwtools import gwtools as _gwtools`, and then you can use the constants as `_gwtools.MSUN_SI`

for example: https://github.com/sxs-collaboration/gwsurrogate/blob/59a18c9a88ec47578aa1aa0539775db7c54fc7b1/gwsurrogate/surrogate.py#L183